front/app/containers/App/constants.ts

```diff
@@ -116,8 +116,8 @@ export const appGraphqlLocalePairs = {
   roRo: 'ro-RO',
   srLatn: 'sr-Latn',
   srSp: 'sr-SP',
-  svSE: 'sv-SE',
-  trTR: 'tr-TR',
+  svSe: 'sv-SE',
+  trTr: 'tr-TR',
```

Comment:
Let's also correct the typo for Turkish while we're here.

Reply:
Nice fix! Did you test that it didn't work before and it works now?

Reply:
(I mean for Turkish)

Reply:
No. I think these changes affect the GraphQL API. The fix to the issue for Swedish that we had detected was the change in `1_locale.rb`.

Reply:
Ok, the change seems reasonable, and these values seem to be used only for typing, so it should be safe to change :+1: